### ISSUE TITLE
post-link.sh doesn't run in windows 64 

### ISSUE THREAD

**Issue by bollwyvl** [Anaconda-Server/nb_config_manager#11]
Installing nb_config_manager into a clean conda on windows 64 will not enable it:
```bat
Z:\> conda create -y -n nbcm python=2.7 && activate nbcm && conda install -y nb_config_manager -c anaconda-nb-extensions

# at this point, the /etc/jupyter/ directory should have been created
[nbcm] Z:\>dir "%CONDA_DEFAULT_ENV%\etc"
 Volume in drive C has no label.
 Volume Serial Number is 2EEC-7DE4

 Directory of C:\cnda\envs\nbcm

File Not Found
```

This causes downstream problems for all extensions that use it.

A local Win-64 build appears to work, so it could just be that re-kicking a CI build with win-64 enabled might do the trick.

**Comment by bollwyvl** [comment]
We may also want to consider adding some tests to see if certain files exist in the correct state after install:
`grep nb_config_manager %PREFIX%etc\jupyter\jupyter_notebook_config.json`.
(or however you'd do that on windows... maybe just `python -c something`, or as a nose test)

This is probably a good thing to do on other extensions/platforms.

**Comment by bollwyvl** [comment]
Fixed by #12, deployed on conda as [0.1.3](https://anaconda.org/anaconda-nb-extensions/nb_config_manager/files?version=0.1.3)
